internal/schemaconverter/oapi3/jsonschema_allow_nulls_test.go

```diff
@@ -764,5 +857,7 @@ func TestGenerateJSONSchemasMapInAReffedObjectAllowingNullValues2(t *testing.T)
 	assert.NoError(t, err)
 	assert.NotNil(t, generatedJSONSchemas)
 	assert.Len(t, generatedJSONSchemas, 2)
-	assert.JSONEq(t, expectedSchema, string(generatedJSONSchemas[1].Bytes))
+	if !assert.JSONEq(t, expectedSchema, string(generatedJSONSchemas[1].Bytes)) {
```

Comment:
This test is still failing (happily in exactly the same way as with oapi2). The generated schema _should_ be an object with an attribute called "object_with_map" (which is a map[string]string).